### PR TITLE
Features/create animals table

### PR DESCRIPTION
**Pull request by fmanimashaun**
# Vet Clinic Database

## Overview

This pull request introduces the initial setup and data for the Vet Clinic database. The database is named `vet_clinic` and includes a table named `animals` with the following columns:

- `id`: integer
- `name`: string
- `date_of_birth`: date
- `escape_attempts`: integer
- `neutered`: boolean
- `weight_kg`: decimal

The SQL used to create this schema can be found in the `schema.sql` file.

## Data

The following data has been inserted into the `animals` table:

1. Agumon: Born on Feb 3rd, 2020, weighs 10.23kg, neutered, no escape attempts.
2. Gabumon: Born on Nov 15th, 2018, weighs 8kg, neutered, escaped 2 times.
3. Pikachu: Born on Jan 7th, 2021, weighs 15.04kg, not neutered, escaped once.
4. Devimon: Born on May 12th, 2017, weighs 11kg, neutered, escaped 5 times.

The SQL used to insert this data can be found in the `data.sql` file.

## Queries

Several queries have been executed against this data:

1. Find all animals whose name ends in "mon".
2. List the name of all animals born between 2016 and 2019.
3. List the name of all animals that are neutered and have less than 3 escape attempts.
4. List the date of birth of all animals named either "Agumon" or "Pikachu".
5. List name and escape attempts of animals that weigh more than 10.5kg.
6. Find all animals that are neutered.
7. Find all animals not named Gabumon.
8. Find all animals with a weight between 10.4kg and 17.3kg.

The SQL used for these queries can be found in the `queries.sql` file.

## Screenshots

Screenshots of the results of these queries have been included in this pull request description.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+/* List the name of all animals born between 2016 and 2019. */
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+
+/* List the name of all animals that are neutered and have less than 3 escape attempts. */
+SELECT name FROM animals WHERE neutered = true AND escape_attempts >= 3;
```
Comment (shahadat3669) [inline queries.sql]: 👏 Great job on your SQL query! However, there's a minor adjustment needed to meet the project requirement:

- [x] As per the project requirement, you should answer the query "List the name of all animals that are neutered and have less than 3 escape attempts." Your current query is close, but it retrieves animals with "neutered" set to true and "escape_attempts" of three or more. To align with the project requirement, kindly consider updating your query to include animals with "neutered" status and less than three escape attempts.

By making this adjustment, your query will accurately address the project requirement. Keep up the great work! 👍🐾📋

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @fmanimashaun, 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ Everything that creates or change the tables structure saved in the schema.sql file.
✅ Everything that populates tables with data saved in the data.sql file.
✅ Added Screenshot in the PR Description.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (fmanimashaun) [reply queries.sql]: SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;

fixed

**Review by gemmen29 — APPROVED** [review]
Hi @fmanimashaun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**